### PR TITLE
CORE-5797 Fix membership unit tests failing to parse certificates from file in some windows environments

### PR DESCRIPTION
**Pull request by charlieR3**

### CODE REVIEW

File: components/membership/registration-impl/build.gradle [hunk 1]
```diff
@@ -41,6 +41,8 @@ dependencies {
 
     implementation "com.typesafe:config:$typeSafeConfigVersion"
 
+    testImplementation "org.apache.commons:commons-text:$commonsTextVersion"
```
Comment (nikinagy) [inline components/membership/registration-impl/build.gradle:44]: why the extra line?

Comment (charlieR3) [reply components/membership/registration-impl/build.gradle:44]: Removed

**Review by nikinagy — APPROVED** [review]
LGTM

**Comment by charlieR3** [comment]
Windows build passed: 
https://ci02.dev.r3.com/job/Corda5/job/Nightlys/job/Corda-Runtime-OS/job/Windows/job/charlie%2Fwindows-test-fix/1/

**Pull request merged**